aws_analysis_tools/cli/instances_2.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+
+#
+# Third party libraries
+#
+
+from texttable import Texttable
+
+#
+# Internal libraries
+#
+
+import krux_boto
+import krux_ec2.cli
+from krux_ec2.filter import Filter
+
+NAME = 'instances'
+
+
+class Application(krux_ec2.cli.Application):
+
+    # A dict with key=CLI options and value=AWS filters
+    _CLI_TO_AWS = {
+            'group': 'group-name',
+            'name': 'tag:Name',
+            'type': 'instance-type',
+            'zone': 'availability-zone',
+            'state': 'instance-state-name'
+        }
+
+    # List of all options
+    _OPTS = ['group', 'name', 'type', 'zone', 'state']
+
+    # A dict with key=CLI options and value=instance attribute
+    _INSTANCE_ATTR = {
+            'group': lambda i: i.groups[0].name,
+            'name': lambda i: i.tags.get('Name'),
+            'type': lambda i: i.instance_type,
+            'zone': lambda i: str(i._placement),
+            'state': lambda i: i.state
+        }
+
+    def __init__(self, name=NAME):
+        # Call to the superclass to bootstrap.
+        super(Application, self).__init__(name=name)
+
+        self.options = vars(self.args)
```

Comment:
Nit: Can you leave a little comment about why you are changing `self.args` into a dictionary?